style.css

```diff
@@ -1 +1,75 @@
 /*  Add your styling here */
+
+.flex-container {
+  display: flex;
+  background-color: #d3d3d3;
+  justify-content: space-around;
+}
+
+.flex-container > div {
+  background-color: #a9a9a9;
+  margin: 0.627em;
```

Comment:
There are some values here  without much context as to why they exist. These are called 'magic numbers'.

It may be useful to add a comment explaining why you used these decimals.